util/multirun.sh

```diff
@@ -35,27 +67,78 @@ fi
 
 cip="$1"
 shift
+args=("$@")
```

Comment:
Why?  The simpler pattern is `for arg` which implies "$@".  E.g.

```
$ function foo() { for arg; do echo ARG: $arg; done }

$ $ foo 1 2 3 
ARG: 1
ARG: 2
ARG: 3

$ foo 1 "2 two" 3 
ARG: 1
ARG: 2 two
ARG: 3
```

Reply:
I didn't know this idiom. Will change.